### ISSUE TITLE
frontend-backend connection: create new schema

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#62]
## User story
1. As a {HR-employee}
2. I want / need {DIdentity to be able to create a new schema}
3. So that {I can use the schema later to request information from DIs}

## Acceptance criteria
* Overview over process:
  * First, HR-employee enters attribute names for new schema in DIdentity frontend #50 
  * Second, backend sends new schema to Lissi-API to create new schema #51 
  * If a success/error-response is received from Lissi-API, a success-/error-message is displayed in the frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
